### PR TITLE
Fix fromGeometry() retain element names for morphTarget & morphNormal

### PR DESCRIPTION
**Pull request by modstorm**
First time contributing, so hopefully I've followed instructions correctly, please let me know if not :)

This fixes element names of morphTargets and morphNormals being lost when converting a Geometry to a GeometryBuffer, using: 

`THREE.BufferGeometry().fromGeometry(geometry)`

I've rebuilt and tested using 'models/animated/horse.js' which does includes morph targets and the names are now included.

### CODE REVIEW

File: src/core/DirectGeometry.js [hunk 1]
```diff
@@ -103,7 +103,7 @@ Object.assign( DirectGeometry.prototype, {
 
 			for ( var i = 0; i < morphTargetsLength; i ++ ) {
 
-				morphTargetsPosition[ i ] = [];
+				morphTargetsPosition[ i ] = { name: morphTargets[i].name, array: [] };
```
Comment (Mugen87) [inline src/core/DirectGeometry.js]: Please add spaces here `morphTargets[i].name`. It should be `morphTargets[ i ].name`

File: src/core/DirectGeometry.js [hunk 2]
```diff
@@ -122,7 +122,7 @@ Object.assign( DirectGeometry.prototype, {
 
 			for ( var i = 0; i < morphNormalsLength; i ++ ) {
 
-				morphTargetsNormal[ i ] = [];
+				morphTargetsNormal[ i ] = { name: morphNormals[i].name, array: [] };
```
Comment (Mugen87) [inline src/core/DirectGeometry.js]: Same here...

**Review by Mugen87 — CHANGES_REQUESTED** [review]

**Comment by sbi-krisj** [comment]
Ok, spacing has been corrected.

File: src/core/DirectGeometry.js (hunk 1 above)
Comment (sbi-krisj) [reply src/core/DirectGeometry.js]: spaces added.

File: src/core/DirectGeometry.js (hunk 2 above)
Comment (sbi-krisj) [reply src/core/DirectGeometry.js]: spaces added.

**Review by Mugen87 — APPROVED** [review]

**Comment by Mugen87** [comment]
Solved via #14711

**Pull request closed without merging**